editor/src/core/model/element-metadata-utils.ts

```diff
@@ -1310,7 +1295,7 @@ export const MetadataUtils = {
         ...element,
         templatePath: newTemplatePath,
         element: newElementInner,
-        children: duplicatedChildren,
+        children: [], // all descendants have new UID-s
```

Comment:
does this break anything on the UI? afaik we create this mock metadata so that things like the navigator can already render the new elements even before the dom report / spy report arrives one frame later.

Reply:
I think the metadata is used to have the updateFrames run correctly in the next step. 
The navigator is hard to decide because when elements change in general it's not so smooth.